tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -403,6 +403,98 @@ public function testUpdateWithSameColumnInDataAndIdentifiers()
         );
     }
 
+    /**
+     * @group DBAL-2688
+     */
+    public function testUpdateWithIsNull()
+    {
+        $driverMock = $this->createMock('Doctrine\DBAL\Driver');
```

Comment:
I copy pasted these from existing test `testUpdateWithSameColumnInDataAndIdentifiers` - I guess the code style has changed since then? I'll get the changes done